playwright.config.ts

```diff
@@ -9,6 +9,7 @@ export default defineConfig({
 		port: DEV ? 5173 : 4173
 	},
 	testDir: 'e2e',
+	timeout: 180000,
```

Comment:
With the default timeout (30s), the tests felt a bit flaky - looks like sometime the wallet initialisation wasn't finishing in time when running tests inside a docker in CI. To be on the safer side, I increased the value.

Reply:
If the tests are flaky, it's better to address the root cause rather than just extending the timeout, as this could lead to future failures. Could you please revert the change and provide a PR that ensures the tests run reliably in the CI environment?

Reply:
Removed the timeout param. I initially added it before I started using `iiPage.waitReady`, so it could be one of the reasons why CI threw a timeout error initially. I will obviously keep my eyes on CI and come back to the topic if this issue appears again.